bootstrap/kubeadm/api/v1alpha3/kubeadmbootstrapconfig_types.go

```diff
@@ -112,6 +112,10 @@ type KubeadmConfigStatus struct {
 	// FailureMessage will be set on non-retryable errors
 	// +optional
 	FailureMessage string `json:"failureMessage,omitempty"`
+
+	// ObservedGeneration is the latest generation observed by the controller.
+	// +optional
```

Comment:
Should this be no longer optional, since we are not setting omitempty? https://github.com/eBay/Kubernetes/blob/master/docs/devel/api-conventions.md#optional-vs-required